libs/network/network.c

```diff
@@ -672,7 +672,8 @@ int netCreateSocket(JsNetwork *net, SocketType socketType, uint32_t host, unsign
   if (sckt<0) return sckt;
 
 #ifdef USE_TLS
-  assert(sckt>=0 && sckt<32);
+  // NEED TO FIX - as socket is assumed to be < 32
+  //assert(sckt>=0 && sckt<32);
```

Comment:
@gfwilliams 
FYI @jumjum123 
The move to esp-idf has changed the number range of the file descriptions. They are not longer < 32 and I believe the assert was due to the sckt number being used in flags.

Can you think of a good way to work around this issue?

Reply:
I've just removed those lines. I believe they were there for TLS when we used a bitfield to store whether the socket was using TLS or not. Since @opichals work on UDP the bitfields were no longer needed \o/

Reply:
Thanks